.github/actions/check_artifact_size/action.yaml

```diff
@@ -115,8 +128,8 @@ runs:
                 console.log(`Artifact size is decreased by ${Math.abs(deltaSize)} (${deltaThreshold}%).`);
               } else {
                 console.log(`Artifact size is increased by ${deltaSize} (${deltaThreshold}%).`);
-                if (deltaThreshold > fileSizeThresholds[file]) {
-                  const threshold = (fileSizeThresholds[file] * 100).toFixed(4);
+                const threshold = (fileSizeThresholds[file] * 100).toFixed(4);
```

Comment:
Nit: I think the code would be a little cleaner/clearer if you didn't use `toFixed` on `deltaThreshold` and `threshold` before comparing them and only do it when printing the percentage below. Then you can also change it to `.toFixed(1)` for simplicity.

Reply:
yea, but that way I'll need to repeat toFixed in both if conditions.
I'll keep it as it is and change it to .toFixed(1) for simplicity.